test/service/Cargo.toml

```diff
@@ -20,10 +20,10 @@ sc-chain-spec = { git = "https://github.com/paritytech/substrate", branch = "pol
 sc-client-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17" }
 sc-transaction-pool-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17" }
 sc-consensus = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17" }
-sc-executor = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17" }
+sc-executor = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17", features = ["wasmtime"] }
```

Comment:
Enabling it on `sc-service` results in the same.

Reply:
So it is redundant to set in both `sc-service` and `sc-executor` only for the test client?
Why is this not the case on the true clients?